test/views/html/cards/LatestNewsAndUpdatesViewSpec.scala

```diff
@@ -45,8 +45,8 @@ class LatestNewsAndUpdatesViewSpec extends ViewSpec {
-          "feature.news.nicSection.start-date" -> "2022-04-01"
+          "feature.news.nicSection.content-en"           -> "<p class=\"govuk-body\">From 6 April 2022 to 5 April 2023 National Insurance contributions will increase by 1.25 percentage points. This will be spent on the NHS, health and social care in the UK.</p><p class=\"govuk-body\">The increase will apply to:</p><ul class=\"govuk-list govuk-list--bullet\"><li>Class 1 (paid by employees)</li><li>Class 4 (paid by self-employed)</li><li>secondary Class 1, 1A and 1B (paid by employers)</li></ul><p class=\"govuk-body\">The increase will not apply if you are over the State Pension age.</p><p class=\"govuk-body govuk-!-margin-bottom-3\"><a href=\"https://www.gov.uk/guidance/prepare-for-the-health-and-social-care-levy\" class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\">Prepare for the Health and Social Care Levy (opens in new tab)</a></p>\"\n      short-description-cy: \"Cynnydd o 1.25 pwynt canrannol yng nghyfraniadau Yswiriant Gwladol\"\n      content-cy: \"<p class=\"govuk-body\">O 6 Ebrill 2022 i 5 Ebrill 2023, bydd cyfraniadau Yswiriant Gwladol yn cynyddu 1.25 pwynt canrannol. Caiff hyn ei wario ar y GIG ac ar iechyd a gofal cymdeithasol yn y DU.</p><p class=\"govuk-body\">Bydd y cynnydd yn berthnasol i’r canlynol:</p><ul class=\"govuk-list govuk-list--bullet\"><li>CYG Dosbarth 1 (a delir gan gyflogeion)</li><li>CYG Dosbarth 4 (a delir gan bobl hunangyflogedig)</li><li>CYG Dosbarth 1, 1A ac 1B eilaidd (a delir gan gyflogwyr)</li></ul><p class=\"govuk-body\">Ni fydd y cynnydd yn berthnasol os ydych dros oedran Pensiwn y Wladwriaeth.</p><p class=\"govuk-body govuk-!-margin-bottom-3\"><a href=\"https://www.gov.uk/guidance/prepare-for-the-health-and-social-care-levy.cy\" class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\">Paratoi ar gyfer yr Ardoll Iechyd a Gofal Cymdeithasol (yn agor tab newydd)</a></p>",
```

Comment:
@pascal22p @SandeepUmredkar @Luka-Abey  Looks like this change shouldn't be part of this ticket. Can you please review this change once again?

Reply:
content-cy is missing main code base file as well 
https://github.com/hmrc/pertax-frontend/blob/main/test/views/html/cards/LatestNewsAndUpdatesViewSpec.scala